docs/HACKING.md

```diff
@@ -160,11 +160,17 @@ commands like this:
 ```
 python infra/helper.py build_image systemd
 python infra/helper.py build_fuzzers --sanitizer memory systemd ../systemd
+python infra/helper.py check_build --sanitizer memory -e ALLOWED_BROKEN_TARGETS_PERCENTAGE=0 systemd
 python infra/helper.py run_fuzzer systemd fuzz-foo
 ```
 
+Another option would be to open a PR and let [CIFuzz](https://google.github.io/oss-fuzz/getting-started/continuous-integration/)
+make sure the fuzz targets can be built using the OSS-Fuzz toolchain.
+It runs `build_image`, `build_fuzzers`, `check_build` and `run_fuzzer` every
+time PRs get updated.
```

Comment:
I think the OSS-Fuzz is covered for the most part but I think `-Dfuzz-tests=true` should be mentioned as well. It can be done later though.

Reply:
It should probably be mentioned in https://github.com/systemd/systemd/blob/main/docs/CODE_QUALITY.md as well.

Reply:
I think I need to add a link to https://oss-fuzz.com/coverage-report/job/libfuzzer_asan_systemd/latest somewhere

Reply:
Yeah, the link to coverage would be good. We should add more tests…